src/iuse_actor.cpp

```diff
@@ -361,6 +361,7 @@ void consume_drug_iuse::load( JsonObject &obj )
     }
     obj.read( "stat_adjustments", stat_adjustments );
     obj.read( "fields_produced", fields_produced );
+    obj.read( "moves", moves);
```

Comment:
`JsonObject::read` won't touch the variable when the JSON data does have that member, that makes it optional in JSON. This may or may not be the intended behavior, *if* you keep it optional, you have to initialize `moves`.

Simplest way to do so is to add it to the declaration of `moves`, or initialize it in the constructor, your choice.

Or you make it mandatory using `moves = obj.get_int( "moves" );`